src/src-schemas/contrl.json.schema

```diff
@@ -0,0 +1,559 @@
+{
```

Comment:
Whats the difference between this and "src/src-schemas/contrl.schema.json" ?

Reply:
One is my version the other is your original version in case I missed something